src/extractor/guidance/roundabout_handler.cpp

```diff
@@ -329,23 +326,13 @@ RoundaboutType RoundaboutHandler::getRoundaboutType(const NodeID nid) const
         if (1 == roundabout_name_ids.size() &&
             0 == connected_names.count(*roundabout_name_ids.begin()))
             return RoundaboutType::Rotary;
-        else
-            return RoundaboutType::Roundabout;
     }
 
-    if (radius <= MAX_ROUNDABOUT_INTERSECTION_RADIUS)
```

Comment:
In this case it is actually a incredibly large loop that really follows roundabout rules. Not exactly sure anymore were it was, but a friend of mine working at a different navigation company told me about this case.